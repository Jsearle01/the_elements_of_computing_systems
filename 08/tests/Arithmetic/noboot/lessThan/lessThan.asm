@SP
M=M-1
A=M
D=M
A=A-1
D=D-M
@LT_TRUE_0
D;JGT
D=0
@LT_END_0
0;JMP
(LT_TRUE_0)
D=-1
(LT_END_0)
@SP
A=M-1
M=D
