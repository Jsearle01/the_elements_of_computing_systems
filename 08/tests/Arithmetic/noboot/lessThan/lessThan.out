|   D    | RAM[0] |RAM[256]|RAM[257]|RAM[258]|
|     -1 |    257 |     -1 |     10 |      0 |
